bundles/best_practices.rst

```diff
@@ -75,7 +76,7 @@ configuration options (see below for some usage examples).
 Directory Structure
 -------------------
 
-The basic directory structure of an AcmeBlogBundle must read as follows:
+The basic directory structure of an ``AcmeBlogBundle`` must read as follows:
```

Comment:
Should we then remove the backticks from the previous section? There are 3 locations where the bundle name is in backticks. ~25% of the references to AcmeBlogBundle on this page are not in backticks while the rest are. (Again, just going for consistency, but don't really care.)